.gitignore

```diff
@@ -75,3 +77,4 @@ clj-kondo*.snap
 .DS_Store
 !corpus/*jar
 lsp
+run();
```

Comment:
why is this needed?

Reply:
My apologies. I think this was an unintended mistake.
Will revert.